README.md

```diff
@@ -1,6 +1,6 @@
-# Project Name
+# Frontdoor terraform script with App Service
 
-This project provides a sample terraform script for provisioning a WAF enabled frontdoor with backend pools set with an existing Azure App service, routing rules with  caching config exposed on a private vnet
+This project provides a end to end terraform script for provisioning a WAF enabled frontdoor with backend pools set with an existing Azure App service, routing rules with  caching config. The script provisions an existing voting App from https://github.com/Azure-Samples/azure-voting-app-redis and limits access to this backend from Azure Frontdoor.
```

Comment:
rewrote this a bit:

This project end to end terraform scripts for provisioning a WAF enabled Azure Front Door with backend pools set with an existing Azure App service, routing rules with caching config. 

These scripts:
1. Provision an Azure Front Door with Web Application Firewall (WAF) enabled
2. Provision a sample application on Azure App Service(Azure Vote - https://github.com/Azure-Samples/azure-voting-app-redis)
3. Configures Front Door to route traffic to the App Service with caching configuration
4. Places limits on inbound traffic to the App Service to be limited to Azure Infrastructure

Reply:
DOne!